packages/ui-components/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "outDir": "dist",
     "lib": [
       "es6",
-      "dom"
+      "dom",
+      "es2019"
```

Comment:
This addition to the config was needed to support the `Object.fromEntries(` in the DownloadVisual component